controllers/mdutil/util.go

```diff
@@ -472,6 +472,21 @@ func GetActualReplicaCountForMachineSets(machineSets []*clusterv1.MachineSet) in
 	return totalActualReplicas
 }
 
+// GetTotalReplicaCountForMachineSets returns sum of max(ms.Spec.Replicas, ms.Status.Replicas)
```

Comment:
```suggestion
// GetTotalReplicaCountForMachineSets returns sum of max(ms.Spec.Replicas, ms.Status.Replicas) across all the machine sets.
```

Reply:
Also, I'm concerned that the name of this new function, GetTotalReplicaCountForMachineSets, is not sufficiently different from GetReplicaCountForMachineSets to know what the difference is between the two functions. Is there a more descriptive name we could use?

(Also, I assume we need to introduce a new function without changing the existing one because ... it would break things elsewhere?)

Reply:
Will work out a name.  Yes the function is used else where.

Reply:
Scratching my head but couldn't find a better one. Any one have any suggestions ?

Reply:
I agree this makes things very confusing, can we either find a better name or using some sort of input struct to filter out machines instead? 

/hold